src/views/Not-Found.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
Rename the file to NotFound.js, JavaScript standards suggest to use camel case or Pascal's case.